docs/layouts/shortcodes/generated/forst_configurable_configuration.html

```diff
@@ -0,0 +1,168 @@
+<table class="configuration table table-bordered">
+    <thead>
+        <tr>
+            <th class="text-left" style="width: 20%">Key</th>
+            <th class="text-left" style="width: 15%">Default</th>
+            <th class="text-left" style="width: 10%">Type</th>
+            <th class="text-left" style="width: 55%">Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td><h5>state.backend.forst.block.blocksize</h5></td>
+            <td style="word-wrap: break-word;">4 kb</td>
+            <td>MemorySize</td>
+            <td>The approximate size (in bytes) of user data packed per block. The default blocksize is '4KB'.</td>
```

Comment:
nit: what does packed mean here? Can we include guidance as to how to set / tune this value.

Reply:
'packed' means that multiple the user's key-values are recorded in one block.

For guidance, please check the `large_state_tuning.md`